cmd/kyma/alpha/enable/module/opts.go

```diff
@@ -1,19 +1,26 @@
 package module
 
 import (
+	"fmt"
 	"time"
 
 	"github.com/kyma-project/cli/internal/cli"
 	"github.com/pkg/errors"
 )
 
+const (
+	customResourcePolicyCreateAndDelete = "CreateAndDelete"
```

Comment:
can you reference the existing v1beta1 API from lifecycle manager here? Its included as an in-code dependency already.